File: 240106/부분 문자열의 개수/number-of-substrings.py
word = input()
word2 = input()
tal = 0 
for i in range(len(word)-1):
    cnt = 0 
    for j in range(len(word2)):

        if word[i+j] == word2[j]:
            cnt+=1
    if cnt == 2:
        tal+=1
print(tal)

File: 240106/부분 문자열의 개수/number-of-substrings.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        String word = sc.next();
        String word2= sc.next();
        int cnt = 0 ;
        for (int i = 0 ; i<word.length()-1; i++){
            int tal = 0 ;
            for(int j = 0 ; j<2; j++){
                if (word.charAt(i+j) == word2.charAt(j)){
                    tal++;
                }
            
            }
            if (tal == 2){
                cnt++;
            }
        }
        System.out.print(cnt);
    }
}